.github/workflows/post-submit.yaml

```diff
@@ -25,9 +29,35 @@ jobs:
 
       - name: Build and push CSV 0.0.1 + latest images for PR merges to main
         if: ${{ github.ref_type != 'tag' }}
-        run: export IMAGE_REGISTRY=quay.io/medik8s && make container-build-and-push
+        run: export IMAGE_REGISTRY=quay.io/medik8s && make container-build-and-push-community
 
       - name: Build and push versioned CSV and images for tags
         if: ${{ github.ref_type == 'tag' }}
         # remove leading 'v' from tag!
-        run: export IMAGE_REGISTRY=quay.io/medik8s && export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && make container-build-and-push
+        run: export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && make container-build-and-push-community
+
+      - name: Create release with manifests
+        if: ${{ github.ref_type == 'tag' }}
+        # https://github.com/marketplace/actions/github-release-create-update-and-upload-assets
+        uses: meeDamian/github-release@2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: true
+          body: |
+            # Node Maintenance Operator ${{ github.ref_name }}
+            
+            ## Notable Changes
+            
+            * TODO
+            ## Release Artifacts
+            
+            ### Images
+            * Operator: quay.io/repository/medik8s/node-maintenance-operator:${{ github.ref_name }}
```

Comment:
Is it "quay.io/repository/medik8s" or "quay.io/medik8s" like in NHC, or it doesn't make any difference?

Reply:
It does make difference.